src/external_task_api_service.ts

```diff
@@ -129,6 +128,43 @@ export class ExternalTaskApiService implements IExternalTaskApi {
     this._publishExternalTaskFinishedMessage(externalTask, successNotificationPayload);
   }
 
+  private async _fetchOrWaitForExternalTasks<TPayloadType>(topicName: string, maxTasks: number, longPollingTimeout: number)
```

Comment:
I would do it like this:

```suggestion
  private async _fetchOrWaitForExternalTasks<TPayloadType>(topicName: string,
  														 maxTasks: number,
  														 longPollingTimeout: number): Promise<Array<ExternalTask<TPayloadType>>> {
```